service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterMapper.java

```diff
@@ -56,8 +68,9 @@ public Encounter mapEncounter(POCDMT000002UK01ClinicalDocument1 clinicalDocument
         encounter.setPeriod(getPeriod(clinicalDocument));
         setServiceProvider(encounter, clinicalDocument);
         setAppointment(encounter, clinicalDocument);
-        setPatient(encounter, clinicalDocument);
+        setSubject(encounter, clinicalDocument);
         setEpisodeOfCare(encounter, clinicalDocument);
+
         return encounter;
     }
 
```

Comment:
setPatient method below seems to unsused ... what is more,  the called mapPatient() method does not receive any argument, so it cause compilation error

Reply:
This error was caused by merge conflict.